docs/docs/installation.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: Installation
+title: Download
 sidebarDepth: 2
-description: >-
-  This article describes various ways to install pomerium
+description: This article describes various ways to install pomerium
 ---
 
-# Installation
+# Releases
 
-## Overview
+Pomerium is shipped in multiple formats and architectures to suit a variety of deployment patterns. There are two binaries:
 
-Pomerium is shipped in multiple formats and architectures to suit a variety of deployment patterns.  There are two binaries:
+- `pomerium` is the primary server component. It is a monolithic binary that can perform the function of any [services mode](/reference/#service-mode).
+- `pomerium-cli` (optional) is a command-line client for working with Pomerium.  Functions include generating service accounts, and acting as an authentication helper for tools like [kubtctl](topics/kubernetes-integration.md).
 
-`pomerium` is the server component.  It is a monolithic binary that can perform the function of any [services mode](/reference/#service-mode), depending on configuration.
 
-`pomerium-cli` is the user component.  It is a similarly monolithic binary handling user facing capabilities such as pomerium service account creation and authentication helper functions.
+[[toc]]
 
-## Pomerium
+
+## Pomerium server
```

Comment:
```suggestion
## pomerium
```

Otherwise, should change `## pomerium-cli` back to something like `Pomerium Client`, which pairs nicely with `Pomerium Server`.  Either way, I think the two titles should share format.